design-system-docs/src/_component_docs/targeted-content.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: component
+title: Targeted Content
+---
+
+The targeted content component allows users to show and hide sections of related content on a page. It can help users scan and process content on a page more effectively.
+
+However, it does hide content from users and it can be easy for users to miss. When using it, you should aim to use **no more than 6** and you shouldn’t have a targeted content on its own.
+
+Because targeted contents hide content, they should always have a clear label/title that helps users understand what is in the section. You can use `h3` within them to break up the content.
+
+## When to use
+
+When content is only relevant to some particular users visiting the page.
+
+## When not to use
+
+They shouldn’t be used to make a page seem shorter, for example: using them to hide large chunks of content.
+If you find yourself using targeted content to do this, think about:
+
+- If you’re repeating content on the page and could consolidate it.
+- Using headings on the page instead.
+- If you need to move some of the content onto another page.
+
+## Examples
+
+### Default variant
+
+<%= render(Shared::ComponentExample.new("targeted_content_default")) %>
+
+### Adviser variant
+
+<%= render(Shared::ComponentExample.new("targeted_content_adviser")) %>
+
+## JavaScript behaviour
+
+Targeted content requires some additional JavaScript behaviour which can be initialised with:
+
+```js
+import initTargetedContent from '@citizensadvice/design-system/lib/targeted-content';
+initTargetedContent();
+```
+
+**Note**: If you are supporting IE 11 or earlier make sure you include an [Element.prototype.closest polyfill](https://www.npmjs.com/package/element-closest).
+
+### Fallback behaviour
+
+If JavaScript either fails or is disabled in the users browser the fallback is to show the content as open. Disable JavaScript for this page to see the fallback behaviour in action.
+
+## Using with Rails
+
+If you are using the `citizens_advice_components` gem you call the component from within a template using:
+
+```rb
+<%%= render(CitizensAdviceComponents::TargetedContent.new(
```

Comment:
Learning question - I'm used to seeing `%` but I'm unsure what `%%` does?

Reply:
In this context it escapes the code sample, otherwise the code will get executed as we're inside a file that understands erb. From the docs here https://www.bridgetownrb.com/docs/template-engines/erb-and-beyond

> If you need to escape an ERB tag (to use it in a code sample for example), use two percent signs: